htdocs/install/mysql/tables/llx_expedition_package.sql

```diff
@@ -12,7 +12,7 @@ create table llx_expedition_package
   size_units        integer,	       -- unit of all sizes (height, width, depth)
   weight            float,	       -- weight
   weight_units      integer,	       -- unit of weight
-  dangerous_goods   smallint          DEFAULT 0, -- 0 = no dangerous goods or 1 = Explosives, 2 = Flammable Gases, 3 = Flammable Liquids, 4 = Flammable solids, 5 = Oxidizing, 6 = Toxic & Infectious, 7 = Radioactive, 8 = Corrosives, 9 = Miscellaneous (see https://en.wikipedia.org/wiki/Dangerous_goods). I'm not sure if just register 0 (no) or 1 (yes) is enough.
+  dangerous_goods   varchar(20), --(comma separated list eg : 3,5,7) 0 = no dangerous goods or 1 = Explosives, 2 = Flammable Gases, 3 = Flammable Liquids, 4 = Flammable solids, 5 = Oxidizing, 6 = Toxic & Infectious, 7 = Radioactive, 8 = Corrosives, 9 = Miscellaneous (see https://en.wikipedia.org/wiki/Dangerous_goods). I'm not sure if just register 0 (no) or 1 (yes) is enough.
```

Comment:
If you change into sql file, you must alos add the migration instruction into the migration file 20.0.0-21.0.0.sql

Reply:
Hello
ok, thanks for the advice I'll do this.